rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -0,0 +1,280 @@
+                    if min_fen_area_b != ZERO.AREA
+                    else 0.0,
+                )
+
+                rotation_expected_b = (
+                    percent_difference >= ACCEPTABLE_FEN_PERCENTAGE_DIFFERENCE
+                )
+
+                return {
+                    "no_of_rmds": no_of_rmds,
+                    "no_of_output_instance": no_of_output_instance,
+                    "rotation_expected_b": rotation_expected_b,
+                    "has_proposed": has_proposed,
+                    "has_user": has_user,
+                    "has_baseline_0": has_baseline_0,
+                    "has_baseline_90": has_baseline_90,
+                    "has_baseline_180": has_baseline_180,
+                    "has_baseline_270": has_baseline_270,
+                    "has_proposed_output": has_proposed_output,
+                    "has_user_output": has_user_output,
+                    "has_basseline_0_output": has_basseline_0_output,
+                    "has_basseline_90_output": has_basseline_90_output,
+                    "has_basseline_180_output": has_basseline_180_output,
+                    "has_basseline_270_output": has_basseline_270_output,
+                }
+
+            def rule_check(self, context, calc_vals=None, data=None):
+                rotation_expected_b = calc_vals["rotation_expected_b"]
+
+                no_of_rmds = calc_vals["no_of_rmds"]
+                no_of_output_instance = calc_vals["no_of_output_instance"]
+
+                has_proposed = calc_vals["has_proposed"]
+                has_user = calc_vals["has_user"]
+                has_baseline_0 = calc_vals["has_baseline_0"]
+                has_baseline_90 = calc_vals["has_baseline_90"]
+                has_baseline_180 = calc_vals["has_baseline_180"]
+                has_baseline_270 = calc_vals["has_baseline_270"]
+
+                has_proposed_output = calc_vals["has_proposed_output"]
+                has_basseline_0_output = calc_vals["has_basseline_0_output"]
+                has_basseline_90_output = calc_vals["has_basseline_90_output"]
+                has_basseline_180_output = calc_vals["has_basseline_180_output"]
+                has_basseline_270_output = calc_vals["has_basseline_270_output"]
+
+                return (
+                    has_user
+                    and has_proposed
+                    and has_baseline_0
+                    and has_proposed_output
+                    and has_basseline_0_output
+                ) and (
+                    (
+                        rotation_expected_b
+                        and has_baseline_90
+                        and has_baseline_180
+                        and has_baseline_270
+                        and has_basseline_90_output
+                        and has_basseline_180_output
+                        and has_basseline_270_output
+                        and no_of_rmds == 5
+                        and no_of_output_instance == 6
```

Comment:
This is reversed, `no_of_rmds == 6 and no_of_output_instance == 5`

Reply:
Addressed